.github/workflows/tests.yml

```diff
@@ -0,0 +1,41 @@
+name: test
+
+on: [push]
+
+jobs:
+  test:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [macos-latest, windows-latest, ubuntu-latest]
+        node-version: [13.x, 12.x, 10.x]
+        webpack-version: [next, 4]
+        exclude:
+          - node-version: 13.x
+            webpack-version: 4
+          - node-version: 12.x
+            webpack-version: 4
```

Comment:
No need exclude

Reply:
Do you know the `exclude` meaning? It is necessary.

Reply:
We need test what our middleware compatibility with webpack@4 + node@12 and webpack@4 + node@13

Reply:
<img width="644" alt="Screen Shot 2020-04-06 at 21 50 17" src="https://user-images.githubusercontent.com/3367801/78560184-a047d380-7850-11ea-94e5-172b850ccca9.png">

So you want to change these tests cases?

Reply:
Yes, we should do it, some deps in theory can break webpack (for example inject globals in node runtime, it is very rare edge case, but better test it) , better to test it